hpOneView/common.py

```diff
@@ -1695,6 +1696,24 @@ def make_alertMap_dict(notes, etag, state='Active', user='None',
     }
 
 
+def make_migration_information(oaIpAddress, oaUsername, oaPassword, vcmUsername, vcmPassword, iloLicenseType='OneView',
```

Comment:
Currently we are considering to drop the support for the dictionary builders on the Python SDK. Almost all of the makers were marked as deprecated on common.py. It is still an open question. Maybe it will be moved to other modules later or even will be removed.